search-commons/src/main/java/no/unit/nva/search2/common/Query.java

```diff
@@ -335,7 +348,7 @@ private URI nextResultsBySortKey(SwsResponse response, Map<String, String> reque
     private void logSearchKeys() {
         logger.info(
             getSearchParameterKeys()
-                .map(Enum::name)
+                .map(ParameterKey::asCamelCase)
```

Comment:
asCamelCase returnerer jo bare key. Burde det ikke ha vært vanlig getter?
For keyen konverteres til camelCase i konstruktøren?